I have added the section "Note:" with an example (#29336)
freeCodeCamp/freeCodeCamp: guide/english/c/if/index.md

-> a is not less than 5!
-> Value of a is : 100
```

> # Note:

Reviewer: Please do not make the note so large:
```suggestion
<strong>Note: </strong>
```

> If there is no curly braces just after the if statement then by default the next immediate line will be considered inside the if block. So if there is only one line of code is required inside the if statement then we don't need to give a curly braces.

Reviewer: Consider rewording this sentence:
```suggestion
If there are no curly braces just after the if statement, then by default, the next immediate line will be considered inside the `if` block. If there is only one line of code required inside the if statement, then you don't need to use curly braces.
```
